modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/type/SubscriptionType.kt

```diff
@@ -2,7 +2,6 @@ package au.com.shiftyjelly.pocketcasts.models.type
 
 enum class SubscriptionType(val label: String) {
     NONE("none"),
-    PLUS("plus"),
-    PATRON("patron");
+    PLUS("plus");
```

Comment:
Why does this have a value for Plus but not for Patron? And a related question: what is the difference between `SubscriptionType` and `SubscriptionTier`. Depending on the answer, it might be good to add a comment or two explaining this.

Reply:
Internal ref: pdeCcb-2fL-p2#comment-2074

I should have clarified it. `SubscriptionType` is being replaced by `SubscriptionTier` on the server but currently, changes are only available in the dev environment. `SubscriptionType` always returns `Plus`. It is present until we fully migrate to `SubscriptionTier`.

I added a comment [here](https://github.com/Automattic/pocket-casts-android/pull/1110/commits/0ceac30567fdc1124282328681bd43cc2e516311).